samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -85,6 +85,16 @@ spec:
           sentinel: sentinel
           redisReplicas: 5
 
+        - name: jwt
+          # JWT signing secret configuration:
```

Comment:
Somewhere it should mention this is optional

Reply:
Fixed